pipeline/data_harmonisation/controller.py

```diff
@@ -40,18 +47,38 @@ def get_commodity_tagged_article():
     return commodity_tagged_article
 
 
-def compute_topic_score(sentiment, topic, id_col='id'):
+def compute_topic_score(pos_sentiment_col, neg_sentiment_col, topic,
+                        id_col='id'):
     original_id = topic[id_col]
-    scored_topic = topic.drop(id_col, axis=1).apply(lambda x: x * sentiment)
+    # scored_topic = topic.drop(id_col, axis=1).apply(lambda x: x * sentiment)
```

Comment:
This is because the compound sentiment is 80% + positive. So I decided to model the positive and negative separately. It also makes more sense as the response from the market are generally assymetric.